### PR TITLE
Audit - images

### PR DESCRIPTION
**Pull request by DominikaK**
Starting off doc audit: https://jira.ez.no/browse/EZP-27852

Covers:
- Improved information order
- Changed heading levels and names for better browseability
- Unified terminology to use "variation" instead of "alias"
- General text polishing

### CODE REVIEW

File: docs/guide/images.md [hunk 1]
```diff
@@ -205,12 +205,12 @@ In addition to [filters exposed by LiipImagineBundle](http://symfony.com/doc/mas
 | geometry/scalepercent        | \[widthPercent, heightPercent\]             | Scales width and height with provided percent values. Proportion will not be kept.                 |
 | geometry/crop                | \[width, height, startX, startY\]           | Crops the image. Result will have provided width/height, starting at provided startX/startY        |
 | border                       | \[thickBorderX, thickBorderY, color=\#000\] | Adds a border around the image. Thickness is defined in px. Color is "\#000" by default.           |
-| filter/noise                 | \[radius=0\]                                | Smooths the contours of an image (`imagick`/`gmagick` only). `radius` is in pixel.                 |
-| filter/swirl                 | \[degrees=60\]                              | Swirls the pixels of the center of an image (`imagick`/`gmagick` only). `degrees` defaults to 60°. |
+| filter/noise                 | \[radius=0\]                                | Smooths the contours of an image (`imagick`/`gmagick` only). `radius` is in px.                    |
```
Comment (mere-teresa) [inline docs/guide/images.md]: Why this change (pixels in px)?

File: docs/guide/images.md [hunk 2]
```diff
@@ -2,11 +2,13 @@
 
 ## Introduction
 
-Image variations (image aliases) allow you to define and use different versions of the same image. You generate variations based on filters which modify aspects such as size and proportions, quality or decorations.
+Image variations (image aliases) enable you to define and use different versions of the same image. You generate variations based on filters which modify aspects such as size and proportions, quality or effects.
```
[the remark this answers is not in the document]
Comment (DominikaK) [reply docs/guide/images.md]: The difference is very slight, but "allow" implies having a permission to do something, while "enable" is a broader ability to do something. E.g. Microsoft Manual of Style suggests not using "allow" in this context.

File: docs/guide/images.md (hunk 1 above)
Comment (DominikaK) [reply docs/guide/images.md]: To be consistent with "px" that is used in the earlier row in the same table.
